docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -173,18 +156,60 @@ Your workspace info should look similar to this:
 
 Lastly, you're going to want to add a _gatsby-config.js_ file to your _site_ directory.
 
-```javascript:title=gatsby-config.js
+```javascript:title=site/gatsby-config.js
 module.exports = {
-  __experimentalThemes: ["theme"],
+  __experimentalThemes: [`theme`],
+}
+```
+
+### Setting up Site `package.json
+
+You will need to add `gatsby` CLI scripts and specify your newly created `theme` as a dependency.
+
+```json:title=site/package.json
+{
+  "name": "site",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    // highlight-start
+    "develop": "gatsby develop",
+    "build": "gatsby build"
+    // highlight-end
+  },
+  "dependencies": {
+    "gatsby": "^2.3.34",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    // highlight-start
+    "theme": "*"
+    // highlight-end
+  }
 }
 ```
 
-While you're still in the _site_ directory, you are going to create an _index.mdx_ file in the pages folder.
+### Run the Site
```

Comment:
Run the site and get some positive feedback before we customize the index page originally exposed in our theme.